### ISSUE TITLE
AssertionError in wrapping/src/test_doc.py

### ISSUE THREAD

**Issue by sagitter** [openmeeg/openmeeg#441]
Hi all.

In Fedora, `openmeeg_python_doc` test is failing with:

```
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/CTest/cmCTestRunTest.cxx:521           Start 1186: openmeeg_python_doc
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/CTest/cmCTestRunTest.cxx:690 
1186: Test command: /usr/bin/python3 "/builddir/build/BUILD/openmeeg-2.4.2/build/wrapping/src/test_doc.py"
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/cmCTest.cxx:306    Current_Time: Mar 05 17:22 UTC
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/CTest/cmCTestRunTest.cxx:772 1186: Test timeout computed to be: 10000000
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/CTest/cmCTestRunTest.cxx:46 1186: Traceback (most recent call last):
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/CTest/cmCTestRunTest.cxx:46 1186:   File "/builddir/build/BUILD/openmeeg-2.4.2/build/wrapping/src/test_doc.py", line 9, in <module>
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/CTest/cmCTestRunTest.cxx:46 1186:     assert inspect.getdoc(om.HeadMat) == headmat_expected_docstring
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/CTest/cmCTestRunTest.cxx:46 1186: AssertionError
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/CTest/cmCTestRunTest.cxx:1062 1186/1188 Test #1186: openmeeg_python_doc ...........................
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/CTest/cmCTestRunTest.cxx:1065 Testing openmeeg_python_doc ... 
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/CTest/cmCTestRunTest.cxx:259 ***Failed    0.14 sec
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/CTest/cmCTestRunTest.cxx:263 Traceback (most recent call last):
  File "/builddir/build/BUILD/openmeeg-2.4.2/build/wrapping/src/test_doc.py", line 9, in <module>
    assert inspect.getdoc(om.HeadMat) == headmat_expected_docstring
AssertionError
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/cmCTest.cxx:306    Current_Time: Mar 05 17:22 UTC
/builddir/build/BUILD/cmake-3.23.0-rc2/Source/CTest/cmCTestMultiProcessHandler.cxx:167 test 1187
```

We are using `Python-3.10.2`

**Comment by papadop** [comment]
Strange because I'm mostly developing openmeeg under fedora (35 for the time being).
Indeed HeadMat changed recently from a class to a simple function.
After investigation, the problem comes from two different things:
- doxy2swig was not handling subnamespaces correctly.
- The string associated to the the HeadMat function needed to be changed.

Should be corrected in the master with commit 13fc8a996a13c30649226f7de1f71c657c17a0e2.

**Comment by papadop** [comment]
Corrected. Thank you for the report !
